### ISSUE TITLE
How do you implement the error window？

### ISSUE THREAD

**Issue by Zakeiswo** [JiaweiZhao-git/DASNet#2]
Hi, bro
Thanks for your work first.
However, I'd like to ask you  a question **that how do you implement the error window**?

**Comment by JiaweiZhao-git** [comment]
# Control the error window size by kernel_size
    logDiff = torch.abs(F.avg_pool2d(torch.log(dpred) - torch.log(depth), kernel_size=7, stride=1, padding=3))
